docs/openapi/components/schemas/credentials/VerifiableBusinessCard.yml

```diff
@@ -37,16 +37,197 @@ properties:
         - VerifiableBusinessCard
   id:
     type: string
+    format: uri
   name:
     type: string
+  description:
+    type: string
   issuanceDate:
     type: string
+    format: date-time
   expirationDate:
     type: string
+    format: date-time
   issuer:
-    $ref: ../common/Organization.yml
+    type: string
+    format: uri
   credentialSubject:
-    $ref: ../common/Organization.yml
+    type: object
```

Comment:
Why not just point to the Person .yml? If we keep going down this path, we will end up never reusing existing schemas

Reply:
we don't need to pull in 100% of what a person is here... it can inflate the required fields, see:

https://w3c-ccg.github.io/traceability-vocab/openapi/components/schemas/common/Person.yml

```yml
$linkedData:
  term: Person
  '@id': https://schema.org/Person
title: Person
description: A person
type: object
properties:
  type:
    type: array
    readOnly: true
    const:
      - Person
    default:
      - Person
    items:
      type: string
      enum:
        - Person
  firstName:
    title: First Name
    description: Person's first name.
    type: string
    $linkedData:
      term: firstName
      '@id': https://schema.org/givenName
  lastName:
    title: Last Name
    description: Person's last name.
    type: string
    $linkedData:
      term: lastName
      '@id': https://schema.org/familyName
  email:
    title: Person's Email Address
    description: Person's email address.
    type: string
    $linkedData:
      term: email
      '@id': https://schema.org/email
  phoneNumber:
    title: Phone Number
    description: Person's contact phone number.
    type: string
    $linkedData:
      term: phoneNumber
      '@id': https://schema.org/telephone
  worksFor:
    title: Works For
    description: Company which employs the person.
    $ref: ./Organization.yml
    $linkedData:
      term: worksFor
      '@id': https://schema.org/worksFor
  jobTitle:
    title: Job Title
    description: Person's job title.
    type: string
    $linkedData:
      term: jobTitle
      '@id': https://schema.org/jobTitle
  taxId:
    title: Tax Identification Number
    description: >-
      The Tax / Fiscal ID of the organization or person, e.g., the TIN in the US
      or the CIF/NIF in Spain.
    type: string
    $linkedData:
      term: taxId
      '@id': https://schema.org/taxID
additionalProperties: false
required:
  - type
example: |-
  {
    "type": ["Person"],
    "firstName": "Amelie",
    "lastName": "Pfeffer",
    "email": "Felton_Hauck15@example.com",
    "phoneNumber": "555-155-2858",
    "worksFor": {
      "type": ["Organization"],
      "name": "Toy - Spinka",
      "description": "Networked human-resource secured line",
      "location": {
        "type": ["Place"],
        "address": {
          "type": ["PostalAddress"],
          "streetAddress": "69628 Gorczany Port",
          "addressLocality": "Joechester",
          "addressRegion": "South Carolina",
          "postalCode": "88805-6256",
          "addressCountry": "Heard Island and McDonald Islands"
        }
      },
      "email": "Greyson15@example.org",
      "phoneNumber": "555-738-4062",
      "faxNumber": "555-109-6587"
    },
    "jobTitle": "Legacy Functionality Associate"
  }
  ```
  
  Person has basically no required fields, so it can't be referred to reliably.

Reply:
In general, optimizing a credential involves cutting down the optional fields, and keeping the `type` consistent, while marking required fields... required.